renderer.php

```diff
@@ -446,12 +467,19 @@ public function course_image_defaults() {
 
         $config = get_config('block_my_courses');
 
+        if (method_exists($this->output, 'image_url')) {
```

Comment:
After evaluation of this statement the code uses $OUTPUT->image_url or $OUTPUT->pix_url depending on whether or not $this->output object has the respective methods. This doesn't make sense.

```
// Hey $this->output. Do you have this function?
if (method_exists($this->output, 'image_url')) {
    // $this->output: I do!

    // Meanwhile we are using a different object?
    $someimage = $OUTPUT->image_url('someimage')
}

// This would make more sense.
if (method_exists($OUTPUT, 'image_url')) {
    // Use the method image_url() in class $OUTPUT.
    $someimage = $OUTPUT->image_url('someimage');
}

// Unless it is $this->output you are wanting to use.
if (method_exists($this->output, 'image_url')) {
    $someimage = $this->output->image_url('someimage');
}
```